scripts/test-all-policy.zeek

```diff
@@ -98,6 +98,7 @@
 @load misc/unknown-protocols.zeek
 @load protocols/conn/community-id-logging.zeek
 @load protocols/conn/disable-unknown-ip-proto-support.zeek
+@load protocols/conn/failed-services.zeek
```

Comment:
OCD: To adhere to the existing extension scripts: `service-violation-logging.zeek`? 

```
community-id-logging.zeek
vlan-logging.zeek
mac-logging.zeek
ip-proto-name-logging.zeek
```
:face_holding_back_tears:

Reply:
As part of addressing some of the last comments, the column was renamed to ``failed_services``. The script is now called ``failed-services-logging``.